rclc_lifecycle/src/rclc_lifecycle/rclc_lifecycle.c

```diff
@@ -265,3 +304,156 @@ rclc_lifecycle_execute_callback(
 
   return (*lifecycle_node->callbacks.fun_ptrs[transition_id])();
 }
+
+rcl_ret_t
+rclc_lifecycle_init_get_state_server(
+  rclc_lifecycle_node_t * lifecycle_node,
+  rclc_executor_t * executor)
+{
+  rclc_lifecycle_service_context_t context;
+  context.lifecycle_node = lifecycle_node;
+
+  rcl_ret_t rc = rclc_executor_add_service_with_context(
+    executor,
+    &lifecycle_node->state_machine->com_interface.srv_get_state,
+    &lifecycle_node->gs_req,
+    &lifecycle_node->gs_res,
+    rclc_lifecycle_get_state_callback,
+    &context);
+  if (rc != RCL_RET_OK) {
+    PRINT_RCLC_ERROR(main, rclc_executor_add_service_with_context);
+    return RCL_RET_ERROR;
+  }
+
+  return rc;
+}
+
+void
```

Comment:
You intentionally want to return a `void` not `rcl_ret_t`  like in the function before?